README.md

```diff
@@ -36,6 +36,14 @@ Initiative (PSI) formats for MS data. The main contributors of OpenMS are
 currently the Eberhard-Karls-Universität in Tübingen, the Freie Universität
 Berlin, and the ETH Zürich.
 
+## All Job Status
```

Comment:
How about putting all this on the OpenMS overview page instead?

".github" repository.

Reply:
Done. Thanks @jpfeuffer .